### PR TITLE
add component test for Hackpack.jsx

### LINKED ISSUE #1392: Create Hackpack Component Test

### PR DESCRIPTION
**Pull request by lowbabun**

### CODE REVIEW

File: src/components/user/Hackpack.jsx [hunk 1]
```diff
@@ -4,18 +4,22 @@ import Link from "next/link";
...
   return (
     <Link
+      data-cy="link-component-test"
       className="w-full bg-gray-100 rounded-xl p-4 hover:border-gray-300 border-gray-100 border-2 duration-300"
       href={link}
       target="_black"
     >
-      <div className="font-bold">{text}</div>
-      <div className="flex flex-wrap">
+      <div data-cy="Header-test" className="font-bold">
```
Comment (shahdivyank) [inline src/components/user/Hackpack.jsx]: tags should all be lowercase

File: src/components/user/Hackpack.jsx [hunk 2]
```diff
@@ -4,18 +4,22 @@ import Link from "next/link";
...
+      </div>
+      <div data-cy="Techs-test" className="flex flex-wrap">
         {techs.map((tech, index) => (
           <div
+            data-cy="tech-variable-component"
             key={index}
             className="text-gray-400 rounded-full flex items-center px-1"
           >
-            <div className="text-hackathon-blue-100 mr-1">
+            <div data-cy="favicon" className="text-hackathon-blue-100 mr-1">
```
Comment (shahdivyank) [inline src/components/user/Hackpack.jsx]: this isn't a favicon, its just can icon

File: src/components/user/Hackpack.jsx [hunk 3]
```diff
@@ -4,18 +4,22 @@ import Link from "next/link";
...
       className="w-full bg-gray-100 rounded-xl p-4 hover:border-gray-300 border-gray-100 border-2 duration-300"
       href={link}
       target="_black"
     >
-      <div className="font-bold">{text}</div>
-      <div className="flex flex-wrap">
+      <div data-cy="Header-test" className="font-bold">
+        {text}
+      </div>
+      <div data-cy="Techs-test" className="flex flex-wrap">
```
Comment (shahdivyank) [inline src/components/user/Hackpack.jsx]: tags should be all lowercase

File: cypress/component/user/Hackpack.cy.jsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import Hackpack from "@/components/user/Hackpack";
+describe("HackPacks", () => {
+  it("HackPack Components", () => {
+    const text = "Hackpack";
```
Comment (shahdivyank) [inline cypress/component/user/Hackpack.cy.jsx]: most of these variables can be placed directly where they r being used, making multiple variables makes it harder to track everything

File: src/components/user/Hackpack.jsx [hunk 5]
```diff
@@ -4,18 +4,22 @@ import Link from "next/link";
 const Hackpack = ({ text, techs, link }) => {
   return (
     <Link
+      data-cy="link-component-test"
```
Comment (shahdivyank) [inline src/components/user/Hackpack.jsx]: data-cy names should be consistent, should all be of the form `hackpack-*` ie `hackpack-link` `hackpack-text`

**Review by shahdivyank — APPROVED** [review]

**Comment by lowbabun** [comment]
should I merge?

**Pull request merged**